werkzeug/serving.py

```diff
@@ -543,7 +553,7 @@ def make_server(host=None, port=None, app=None, threaded=False, processes=1,
     elif threaded:
         return ThreadedWSGIServer(host, port, app, request_handler,
                                   passthrough_errors, ssl_context, fd=fd)
-    elif processes > 1:
+    elif processes > 1 and can_fork:
```

Comment:
@davidism Yeah, which exception is most appropriate? `OSError`?

And should this logic remain here, or go inside the `ForkingWSGIServer` class initialization?